src/App.js

```diff
@@ -0,0 +1,85 @@
+import { createElement } from "react";
+import ReactDOM from "react-dom/client";
```

Comment:
have a empty line between library imports and local file imports for better readability